sdk/providers.md

```diff
@@ -0,0 +1,162 @@
+# Providers
+
+For information on installing and using the SDK, please begin by reading the [getting started guide](getting-started.md).
```

Comment:
we should talk about what is a provider and why is it used

Reply:
Where can I find this information?